tests/test_mutualauth.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import unittest
+
+import falcon
+from falcon_mutualauth.mutualauth import Authorize
+
+
+class FakeRequest(object):
+
+    def __init__(self, role):
+        self.__role = role
```

Comment:
Is this meant to signal that `role` should be a private instance variable? If so, a single underscore is conventional :).